CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [1.4.0] - 2020-09-21
```

Comment:
This is heads up that the feature will be tested by the testers, then it can only be released to vscode market. So we will change the date in future!

Thanks for contributing this!